### ISSUE TITLE
Stringify UUIDs

### ISSUE THREAD

**Issue by rtrox** [rtrox/mssqlcli#60]
UUIDs will need to be normalized to strings:
```
Code/mssqlcli [ mssqlcli -o json query ~/sql_queries/show_databases.sql                                                            master * ] 9:31 AM
Traceback (most recent call last):
  File "/Users/russ7612/Code/mssqlcli/venv/bin/mssqlcli", line 9, in <module>
    load_entry_point('mssqlcli==1.0.1b0', 'console_scripts', 'mssqlcli')()
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/click-6.6-py2.7.egg/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/click-6.6-py2.7.egg/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/click-6.6-py2.7.egg/click/core.py", line 1060, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/click-6.6-py2.7.egg/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/click-6.6-py2.7.egg/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/click-6.6-py2.7.egg/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/mssqlcli-1.0.1b0-py2.7.egg/mssqlcli/cli.py", line 83, in query
    output = ctx.obj['output_formatter'](results)
  File "/Users/russ7612/Code/mssqlcli/venv/lib/python2.7/site-packages/mssqlcli-1.0.1b0-py2.7.egg/mssqlcli/formats.py", line 57, in jsonify
    formatted_json = json.dumps(stringify({"results": obj}), indent=4)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 209, in encode
    chunks = list(chunks)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 434, in _iterencode
    for chunk in _iterencode_dict(o, _current_indent_level):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 408, in _iterencode_dict
    for chunk in chunks:
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 332, in _iterencode_list
    for chunk in chunks:
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 408, in _iterencode_dict
    for chunk in chunks:
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 442, in _iterencode
    o = _default(o)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/json/encoder.py", line 184, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: UUID('00000000-0000-0000-0000-000000000000') is not JSON serializable
```

should be accomplished in the ```stringify``` function in formats.py

Steps to reproduce:
    1. Use database with UUID fields.
    2. attempt to perform any query with ```-o json```

Only affects JSON, CSV and Pretty Print function as expected.

**Comment by sulami** [comment]
I'd be happy to take this over

**Comment by rtrox** [comment]
Apologies sulami! I just made PR #63 before I saw your comment. I'll merge rather than mine if you'd still like to fix this, just let me know.

Thanks!

**Comment by sulami** [comment]
Ah, no problem

**Comment by rtrox** [comment]
Closed via #63
